templates/console/source/lambda/config-version/cf_config_version_manager/cf_config_version_manager.py

```diff
@@ -544,77 +583,86 @@
     data = response['Item']
     if not data:
         raise Exception(f"Failed to get the version with distribution id:{distribution_id}, version_id:{src_version}")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using `Exception` and `BaseException` can make your code prone to errors and difficult to maintain. Instead, we recommend using one of the [Built-in Exceptions](https://docs.python.org/3/library/exceptions.html) or creating a custom exception class that is derived from `Exception` or one of its subclasses.